Code/Tools/ProjectManager/Source/GemCatalog/GemFilterWidget.cpp

```diff
@@ -428,6 +471,12 @@ namespace O3DE::ProjectManager
         {
             m_filterProxyModel->SetGemActive(GemSortFilterProxyModel::GemActive::NoFilter);
         }
+
+        // Missing
+        if (button == buttons[4])
```

Comment:
The above uses of `buttons<SomeInt>` all stored it in a variable which helps document what it was for
```
QAbstractButton* activeButton = buttons[2];
QAbstractButton* inactiveButton = buttons[3];
```

I also recommend doing that for `buttons[4]`
```suggestion
        const QAbstractButton* missingGemButton = buttons[4];
        if (button == missingGemButton)
```